src/utilities/__tests__/getNumericalTarget.test.ts

```diff
@@ -0,0 +1,66 @@
+import { getClasses } from '../getClasses';
+import { getClassesPairs } from '../getClassesPairs';
+import { getNumericalTarget } from '../getNumericalTarget';
+import { getSelectedResults } from '../getSelectedResults';
+
+const target = [
+  'class1',
+  'class1',
+  'class1',
+  'class1',
+  'class2',
+  'class2',
+  'class2',
+  'class2',
+  'class3',
+  'class3',
+  'class3',
+  'class3',
+];
+
+const predicted = [
+  0.95, 0.15, 0.13, 0.08, 0.93, 0.91, 1.99, 0.12, 1.85, 1.95, 1.75, 1.99,
+];
+
+describe('Get numerical targets', () => {
+  it('Get targets', () => {
+    const classes = getClasses(target);
+    const pairs = getClassesPairs(classes);
+    let result: number[][] = [];
+    for (let pair of pairs) {
+      const numericalTarget = getNumericalTarget(
+        getSelectedResults(target, pair),
+        getSelectedResults(predicted, pair),
+        pair,
+      );
+      result.push(numericalTarget);
+    }
+
+    expect(result[0][0]).toBeCloseTo(0.079);
```

Comment:
Using https://www.npmjs.com/package/jest-matcher-deep-close-to you can directly expect(result) to the matrice.